meteor/client/ui/SegmentTimeline/Renderers/VTSourceRenderer.tsx

```diff
@@ -466,11 +466,13 @@ export class VTSourceRendererBase extends CustomLayerItemRenderer<IProps & WithT
 			<span className="segment-timeline__piece__label" ref={this.setLeftLabelRef} style={this.getItemLabelOffsetLeft()}>
 				{noticeLevel !== null && <PieceStatusIcon noticeLevel={noticeLevel} />}
 				<span
-					className={ClassNames('segment-timeline__piece__label', {
+					className={ClassNames('segment-timeline__piece__label', 'with-duration', {
 						'overflow-label': end !== '',
+						'swap-duration': this.shouldSwapDurationLabel(),
```

Comment:
Might fit better with existing patterns (e.g. header components) to have `duration-left`, `duration-right` or even `with-duration.left` and `with-duration.right` as the CSS rules

Reply:
Done